src/tests/youtube/Youtube_EME_001.test.js

```diff
@@ -0,0 +1,119 @@
+import {
+  pluginDeactivate,
+  webKitBrowserStartAndResume,
+  setWebKitUrl,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+import { AttachToLogs } from '../../commonMethods/remoteWebInspector'
+
+let url =
+  'http://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2016.html?test_type=encryptedmedia-test&enablewebm=off&command=run&tests=1,6,17,18'
```

Comment:
Needs to be updated to the 2018 or 2019 eme tests: http://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2018.html?test_type=encryptedmedia-test&enablewebm=off&command=run&timestamp=1582733202787

or: http://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2019.html?test_type=encryptedmedia-test&enablewebm=off&command=run&timestamp=1582733202787

Reply:
Hi @wouterlucas ,
In metromatic test, only 1,6,7,18 tests were run. Please let me know what tests to be run here.